src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/FormStore.js

```diff
@@ -109,7 +119,42 @@ export default class FormStore {
         return this.resourceStore.data;
     }
 
-    save() {
+    @action validate() {
+        const {validator} = this;
```

Comment:
I've tried, and flow complains because the validator might be null. We could change the line to `const validator = this.validator` instead, if you like that better.